app/models/album.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const User = require('../models').users;
+
+module.exports = (sequelize, DataTypes) => {
+  var album = sequelize.define('albums', {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
```

Comment:
If you set this id (which represents the albumId) as the only primaryKey, then only one user would be able to buy a given album A, but the idea is that many users could buy the album A.